mybooks.sql

```diff
@@ -0,0 +1,64 @@
+-- phpMyAdmin SQL Dump
+-- version 5.0.2
+-- https://www.phpmyadmin.net/
+--
+-- Hôte : 127.0.0.1:3306
+-- Généré le : jeu. 05 nov. 2020 à 13:48
+-- Version du serveur :  5.7.31
+-- Version de PHP : 7.4.9
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Base de données : `mybooks`
+--
+CREATE DATABASE mybooks;
+-- --------------------------------------------------------
+USE mybooks;
```

Comment:
You probably should let the user choose the name of his·er database to avoid name conflicts

Reply:
Done